interface/src/componentsStyling/encrypt.scss

```diff
@@ -1,76 +1,144 @@
 @import './theme.scss';
 
-.form-container {
+.big-box-shadow {
     @include themed() {
-        form {
-            padding-top: 15px;
-            .MuiFormControl-root {
-                width: 100%;
-                .MuiOutlinedInput-root {
-                    margin: 5px 0;
-                    input {
+        top: 238px;
+        left: 462px;
+        width: 1414px;
+        height: 635px;
+        background: t('box-bg');
+        box-shadow: 4px 0px 11px t('box-shadow');
+        border-radius: 25px 10px 85px 25px;
+        opacity: 1;
+        margin-left: 25px;
+    }
+}
+
+.main-container {
+    @include themed() {
+        .form-container {
+            background: t('box-bg');
+            box-shadow: t('box-shadow');
+            border-radius: 25px 10px 85px 25px;
+            margin-bottom: 35px;
+            form {
+                padding-top: 15px;
+                position: relative;
+                .MuiFormControl-root {
+                    width: 100%;
+                    .MuiOutlinedInput-root {
+                        margin: 5px 0;
+                        input {
+                            box-sizing: inherit;
+                            font-size: 12px;
+                            color: t('text');
+                            background-color: t('text-field-color');
+                            border: 1px solid t('text-field-color');
+                            border-radius: 32px;
+                            height: 50px;
+                            left: 489px;
+                            margin-bottom: 40px;
+                        }
+                        fieldset {
+                            display: none;
+                        }
+                    }
+                    .MuiInputBase-multiline {
                         box-sizing: inherit;
                         font-size: 12px;
                         color: t('text');
                         background-color: t('bg');
                         border: 1px solid t('text');
                         border-radius: 4px;
                     }
-                    fieldset {
-                        display: none;
+                }
+                .address-list {
+                    display: flex;
+                    flex-direction: column;
+                    padding: 5px;
+                    div.MuiButtonBase-root {
+                        margin: 2px 0;
+                        background-color: t('bg');
+                        color: t('text');
+                        border: 1px solid t('text');
+                    }
+                } 
+                .estimated-reward {
+                    top: 933px;
+                    left: 514px;
+                    width: 255px;
+                    height: 23px;
+                    text-align: left;
+                    font: normal normal normal 20px/23px Karla;
```

Comment:
Please verify the css before you are doing copy paste from XD, we don't need all the thing from XD.